packages/bot/src/bot.ts

```diff
@@ -1018,6 +1038,26 @@ const buildBotActions = (client: ClientV2, viemClient: ViemClient, spaceDapp: Sp
         }
     }
 
+    /**
+     * Ban a user from a space
+     * Requires Permission.ModifyBanning to execute this action
+     */
+    const ban = async (userId: string, spaceId: string) => {
+        const tx = await spaceDapp.banWalletAddress(spaceId, userId, client.wallet)
+        const receipt = await tx.wait()
+        return { txHash: receipt.transactionHash }
+    }
+
+    /**
+     * Unban a user from a space
+     * Requires Permission.ModifyBanning to execute this action
+     */
+    const unban = async (userId: string, spaceId: string) => {
+        const tx = await spaceDapp.unbanWalletAddress(spaceId, userId, client.wallet)
+        const receipt = await tx.wait()
+        return { txHash: receipt.transactionHash }
+    }
```

Comment:
note that we're using the client.wallet to pay the gas

maybe in a v2 or something we can expose only the tx and allow the user to pass the desired wallet to pay